app/src/main/kotlin/com/wire/android/ui/common/SearchBar.kt

```diff
@@ -34,7 +34,7 @@ fun SearchBarUI(placeholderText: String, modifier: Modifier = Modifier, onTextTy
     var text by remember { mutableStateOf(TextFieldValue()) }
 
     WireTextField(
-        modifier = modifier.padding(bottom = 16.dp, end = 16.dp, start = 16.dp),
+        modifier = modifier.padding(bottom = 16.dp, end = 8.dp, start = 8.dp),
```

Comment:
Why this change if UserProfileScreen doesn't implement any search?

Reply:
honestly: i don't know why it's here, it should be in `develop` long time ago. Maybe some merging issue, but this is how it actually should be